index.html

```diff
@@ -348,6 +348,936 @@
 ,{"system":"Citus","date":"2022-07-01","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"","tags":["C","PostgreSQL compatible","column-oriented"],"load_time":1579,"data_size":18980918899,"result":[[7.58503,6.70447,6.52499],[6.33941,5.06063,5.00238],[11.7488,9.86417,9.93223],[12.6306,9.36305,9.17061],[40.6101,39.0803,38.1187],[117.654,113.912,113.441],[10.3404,8.08936,7.70732],[6.31542,4.72821,4.72989],[82.5425,77.2124,76.9219],[91.1776,83.4492,82.4727],[14.5474,10.0815,10.3873],[15.4899,11.2922,11.1877],[19.9794,15.5002,17.4492],[76.9216,72.5172,72.7915],[21.5446,17.5691,18.561],[56.9438,54.6387,53.5745],[75.0977,69.7842,70.0259],[31.3299,27.0267,26.3216],[129.417,122.956,121.182],[3.73386,2.14148,2.12737],[34.6021,27.9727,28.6878],[37.152,29.6193,29.2966],[52.2157,37.8589,37.6994],[181.955,149.08,148.471],[15.4687,11.3138,10.3856],[10.2779,8.46868,8.8324],[14.4687,10.4076,11.4263],[47.009,40.2969,39.6888],[749.946,742.979,744.461],[69.4383,67.5636,67.2128],[27.0317,21.4008,20.9524],[36.6675,25.6347,26.4408],[140.424,130.546,129.738],[106.959,92.033,90.1609],[110.98,94.4787,96.2656],[64.4474,60.1853,60.6816],[6.17549,6.25376,5.87004],[1.99153,1.81776,1.80596],[1.00141,0.800271,0.801975],[7.91778,7.70928,8.33299],[0.929845,0.642076,0.638478],[0.866536,0.683567,0.680218],[0.937823,0.784747,0.765929]],"source":"citus/results/c6a.4xlarge.json"}
+,{"system":"ClickHouse Cloud (aws)","date":"2025-01-03","machine":"192GB","cluster_size":"serverless","comment":"","tags":["C++","column-oriented","ClickHouse derivative","managed","aws"],"load_time":185.205,"data_size":9944101793,"result":[[0.003,0.002,0.002],[0.41,0.012,0.307],[0.033,0.195,0.033],[0.053,0.361,0.053],[0.247,0.273,0.245],[0.499,0.626,0.483],[0.132,0.128,0.017],[0.017,0.157,0.017],[0.617,0.749,0.57],[0.465,0.713,0.472],[0.319,0.233,0.231],[0.373,0.268,0.303],[0.636,0.369,0.365],[0.573,0.614,0.658],[0.502,0.681,0.48],[0.277,0.287,0.265],[1.109,1.074,1.165],[0.845,0.855,0.816],[3.34,2.454,2.189],[0.019,0.02,0.019],[0.661,2.656,0.709],[1.999,0.722,0.75],[0.899,3.288,1.877],[2.393,7.441,5.378],[0.201,0.207,0.199],[0.145,0.143,0.141],[0.214,0.204,0.224],[0.671,0.668,0.659],[6.282,6.173,6.344],[0.04,0.039,0.039],[0.426,0.391,0.401],[0.847,0.538,0.667],[1.701,1.908,1.727],[1.823,2.083,1.895],[1.822,1.784,1.846],[0.243,0.244,0.217],[0.044,0.042,0.037],[0.025,0.022,0.025],[0.029,0.026,0.025],[0.068,0.068,0.069],[0.017,0.015,0.016],[0.017,0.014,0.015],[0.012,0.012,0.011]],"source":"clickhouse-cloud/results/aws.192GB.json"}
```

Comment:
No need to update index.html manually. The repository has a CI job which refreshes the website automatically once the PR is merged (based on the .json result file).

Reply:
Thank you - I was referring to older instructions which did require a manual refresh.